RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterQuickFixHelper.cs

```diff
@@ -0,0 +1,195 @@
+﻿using Antlr4.Runtime;
+using Antlr4.Runtime.Tree;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Rubberduck.Inspections.QuickFixes
+{
+    class AssignedByValParameterQuickFixHelper
+    {
+        private Declaration _target;
+        private QualifiedSelection _selection;
+        private HashSet<string> _namesInUse;
+        private ICodeModule _module;
+        public AssignedByValParameterQuickFixHelper(Declaration target, QualifiedSelection selection)
```

Comment:
My hope is that there may be some functionality in this class that is useful on a more general basis.  I'm a new contributor, so I am exceedingly hesitant to propose a system-wide helper class.  So, I intentionally named this class specific to the QuickFix(es) it supports...That said, if a class with similar functionality does not already exist, and other (more experienced) contributors believe there is merit to making some or all of this class more general...that would be great.